niap-cc/Permissions/TransactIds/app/build.gradle

```diff
@@ -18,8 +18,9 @@ apply plugin: 'com.android.application'
 
 android {
     namespace 'com.android.certifications.niap.permissions.transactids'
-    compileSdk 34
-
+    compileSdk 36
```

Comment:
Since this release of the tester is targeting the Android V / SDK version 35 release, should the compileSdk be set to 35 to avoid requiring other users downloading an SDK version that isn't yet (or will only be recently) released?

Reply:
ok